ag/ag.proto

```diff
@@ -152,7 +152,8 @@ message Assignment {
     uint32 order = 7;
     bool isGroupLab = 8;
     uint32 scoreLimit = 9;  // defines minimal score limit for auto approval
-    repeated Submission submissions = 10; 
+    uint32 containerTimeout = 10;
```

Comment:
Will this mess up the DB?

Reply:
Never did for me, the gorm's automigrate always took care of that, but I see your concern. Will change to preserve the old numbering.

Reply:
If auto migrate works, then I'm not very concerned 👍